src/main/java/com/cloudbees/jenkins/plugins/bitbucket/hooks/PullRequestHookProcessor.java

```diff
@@ -83,6 +83,8 @@ public void process(final HookEventType hookEvent, String payload, final Bitbuck
                         eventType = SCMEvent.Type.CREATED;
                         break;
                     case PULL_REQUEST_DECLINED:
+                        eventType = SCMEvent.Type.REMOVED;
+                        break;
```

Comment:
Yes. Nothing new.  It's the same as PULL_REQUEST_MERGED

Reply:
So, why add these two lines?

Reply:
It should trigger pipeline rine in the same way when a pipeline is merged.
Here are 2 screens:
PR-3 was merged, and 2nd pipeline run was triggered. 
![image](https://user-images.githubusercontent.com/16893704/94209583-8b293480-fec4-11ea-8efc-e2114dfa2f78.png)

PR-2 was declined and the pipeline run wasn't triggered. 
![image](https://user-images.githubusercontent.com/16893704/94209615-9e3c0480-fec4-11ea-9a4b-3c3e64f96cdd.png)

The only difference I found is in this `switch/case`. 

To get the same behaviour for declined events as merged I added these lines.

![image](https://user-images.githubusercontent.com/16893704/94210076-b4969000-fec5-11ea-8e80-75376ef10f66.png)

Reply:
Sorry, let me ask this a different way.  It looks to me like the behavior of the code is the same before and after this change.

Before:
```
            switch (hookEvent) {
                case PULL_REQUEST_DECLINED:
                case PULL_REQUEST_MERGED:
                        eventType = SCMEvent.Type.REMOVED;
                        break;
```

After:
```
            switch (hookEvent) {
                case PULL_REQUEST_DECLINED:
                        eventType = SCMEvent.Type.REMOVED;
                        break;
                case PULL_REQUEST_MERGED:
                        eventType = SCMEvent.Type.REMOVED;
                        break;
```

Since these are functionally the same, what does this PR accomplish?